pkg/ovs/openflow/ofctrl_bridge_test.go

```diff
@@ -85,3 +86,34 @@ func TestOFBridgeIsConnected(t *testing.T) {
 	}()
 	wg.Wait()
 }
+
+func TestDeleteGroup(t *testing.T) {
+	b := NewOFBridge("test-br", GetMgmtAddress(ovsconfig.DefaultOVSRunDir, "test-br"))
+
+	for _, m := range []struct {
+		name           string
+		existedGroupID int
+		deleteGroupID  int
+		err            error
+	}{
+		{
+			name:           "delete existed group without flow",
+			existedGroupID: 20,
+			deleteGroupID:  20,
+			err:            nil,
+		},
+		{
+			name:           "delete non-existed group",
+			existedGroupID: 20,
+			deleteGroupID:  30,
+			err:            nil,
+		},
+	} {
+		t.Run(m.name, func(t *testing.T) {
+			b.ofSwitch = newFakeOFSwitch(b)
+			b.ofSwitch.NewGroup(uint32(m.existedGroupID), ofctrl.GroupAll)
```

Comment:
why it doesn't use `b.CreateGroup` to be symmetrical

Reply:
updated